loky/process_executor.py

```diff
@@ -673,14 +672,14 @@ def is_shutting_down():
             _shutdown_crash(executor_flags, processes, pending_work_items,
                             call_queue, cause_msg)
             return
-        executor = executor_reference()
         if (is_shutting_down() and
```

Comment:
`is_shutting_down` is a closure on the `executor` variable so we need to dereference the weak reference `executor_reference()` before calling `is_shutting_down()`.